plugins/BEdita/Core/src/Model/Action/SaveEntityAction.php

```diff
@@ -47,8 +47,10 @@ protected function initialize(array $config)
      */
     public function execute(array $data = [])
     {
-        $entity = $this->Table->patchEntity($data['entity'], $data['data']);
-        $success = $this->Table->save($entity);
+        $entityOptions = !empty($data['entityOptions']) ? (array)$data['entityOptions'] : [];
+        $saveOptions = !empty($data['saveOptions']) ? (array)$data['saveOptions'] : [];
```

Comment:
`(array)Hash::get($data, 'saveOptions')` makes Scrutinizer think this method has lower cyclomatic complexity. 🙃